omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,256 @@
+                                        <label>
+                                            <span class="required">*</span>
+                                            Parent Health facility
+                                        </label>
+                                        <obs id="parent-health-facility" conceptId="166453" />
+                                        <obs id="district-of-parent-health-facility" labelText="District of Parent Health Facility" conceptId="166454" />
+                                        <label>
+                                            <span class="required">*</span>
+                                            Serial No
+                                        </label>
+                                        <obs id="serial-number" conceptId="1646" />
+                                        <label>Encounter Date</label>
+                                        <encounterDate default="today" disallowMultipleEncountersOnDate="block" />
+                                    </div>
+
+                                    <div class="row">
+                                        <div class="col-md-12">
+                                            <obs id="reason-for-visit-other" labelText="Other Specify" conceptId="164975" />
+                                            <obs id="other-regimen" labelText="Other Regimen" conceptId="99126" />
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-body">
+                                <div class="row">
+                                    <div class="col-md-3">
+                                        <label>
+                                            <span class="required">*</span>
+                                            Telephone No
+                                        </label>
+                                        <obs id="telephone-number" conceptId="90266" />
+                                        <label>
+                                            <span class="required">*</span>
+                                            PT Clinic No
```

Comment:
@ODORA0 Change PT Clinic No to ART Number at Parent Facility

Reply:
@ssmusoke Done

Reply:
@ODORA0 This should be an Obs not an identifier, I see it loading the ART Clinic number - look at the comments, same to the EID number - as these are from the parent facility

Reply:
@ssmusoke Done